cmd/cayley/cayley.go

```diff
@@ -204,6 +207,25 @@ func main() {
 
 		handle.Close()
 
+	case "dump":
+		handle, err = db.Open(cfg)
+		if err != nil {
+			break
+		}
+		if !graph.IsPersistent(cfg.DatabaseType) {
```

Comment:
That was actually the very intention I had in mind. This is just food for thought but in the future it could be expended so something along those lines would be possible:

```./cayley dump -db="memstore" -quads="data.nq" -dump="subset.nq" -dump_type="nquad" -dump_query="g.V("charlie").Out("follows")"```

Not sure how much that's actually useful in practice but I could definitely see it come in handy on various occasions.